package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "notreallyapackage",
+  "version": "0.0.0",
+  "description": "not really a package; just using it for the linting",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+  },
+  "devDependencies": {
+    "husky": "^0.14.3",
+    "lint-staged": "^7.0.0",
+    "prettier": "^1.11.1"
+  },
+  "scripts": {
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": ["prettier --write", "git add"]
```

Comment:
Oh, you'll also want to add the js.es6 files to this configuration

Reply:
👍